ml-agents/mlagents/trainers/components/bc/model.py

```diff
@@ -0,0 +1,107 @@
+import tensorflow as tf
+import numpy as np
+from mlagents.trainers.models import LearningModel
+
+
+class BCModel(object):
+    def __init__(self, policy_model: LearningModel, lr, anneal_steps):
+        """
+        Tensorflow operations to perform Behavioral Cloning on a Policy model
+        :param policy_model: The policy of the learning algorithm
+        :param lr: The initial learning Rate for behavioral cloning
+        :param anneal_steps: Number of steps over which to anneal BC training
+        """
+        self.z_size = 128
+        self.alpha = 0.0005
+        self.mutual_information = 0.5
```

Comment:
Is this used? Only see it used on the GAILModel

Reply:
Should these be options in the constructor?

Reply:
Yep. removed!